### ISSUE TITLE
Folders not working

### ISSUE THREAD

**Issue by Goofmann** [emfmesquita/beyondhelp#38]
On the character page, previously created folders still exist and are operational. When trying to create a new folder, I am able to type in a folder name but when the window closes after clicking create folder nothing happens.

**Comment by emfmesquita** [comment]
You are right, this is a bug. Just verified it. Try reloading the page, the folder should show. I still have a bunch of fixes to do. The last two weeks were crazy on my side. I hope to have some time for BH before the weekend.

Thanks for reporting that.
